plugins/connection/podman.py

```diff
@@ -148,10 +153,11 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
 
         # shlex.split has a bug with text strings on Python-2.6 and can only handle text strings on Python-3
         cmd_args_list = shlex.split(to_native(cmd, errors='surrogate_or_strict'))
+        exec_args_list = ["exec"]
         if self.user:
-            cmd_args_list += ["--user", self.user]
+            exec_args_list.extend(("--user", self.user))
```

Comment:
I would think it would be better to prepend to cmd_args_list.

Basically the OPTIONS have to preceed the Container ID>

Reply:
I had the same idea before but the container name comes before `cmd_args_list` and podman exec does not like flags after container name.

```
podman exec [flags] CONTAINER [COMMAND...
```

Reply:
Ok I misunderstood the what this was doing.